src/legacy/fetch.ts

```diff
@@ -75,12 +75,13 @@ export const checkedFetch = (request: FetchRequest) => {
       return response
     })
 }
-export const fetchDefault = (
+export const fetchDefault = <T extends typeof _checkResponse>(
```

Comment:
This is later used to assign the correct type to the response by parsing the response in `checkResponse`